package.json

```diff
@@ -100,14 +100,7 @@
     "test:nuts": "nyc mocha \"**/*.nut.ts\" --slow 4500 --timeout 600000 --parallel",
     "version": "oclif-dev readme"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "sf-husky-pre-commit",
-      "commit-msg": "sf-husky-commit-msg",
```

Comment:
why was this husky section removed?

Reply:
So that I could push code to GH for this initial copy/paste. Linting and tests are failing in this PR. I have a note to add it back.